plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -991,10 +722,10 @@ public ConnectorInsertTableHandle beginRefreshMaterializedView(ConnectorSession
                 .toArray(Type[]::new);
 
         AppendFiles appendFiles = transaction.newFastAppend();
+        FileIO fileIO = transaction.table().io();
         for (CommitTaskData task : commitTasks) {
-            HdfsContext context = new HdfsContext(session);
             DataFiles.Builder builder = DataFiles.builder(icebergTable.spec())
-                    .withInputFile(new HdfsInputFile(new Path(task.getPath()), hdfsEnvironment, context))
+                    .withInputFile(fileIO.newInputFile(task.getPath()))
```

Comment:
same as above

Reply:
Same, this seems to be a rebase conflict and can be removed